TLM/TLM/Util/VersionUtil.cs

```diff
@@ -14,30 +14,37 @@ namespace TrafficManager.Util {
     [SuppressMessage("Performance", "HAA0601:Value type to reference type conversion causing boxing allocation", Justification = "Not performance critical")]
     [SuppressMessage("Performance", "HAA0302:Display class allocation to capture closure", Justification = "Not performance critical")]
     public static class VersionUtil {
+        public const string BRANCH =
 #if LABS
-        public const string BRANCH = "LABS";
+            "LABS";
 #elif DEBUG
-        public const string BRANCH = "DEBUG";
+            "DEBUG";
 #else
-        public const string BRANCH = "STABLE";
+            "STABLE";
 #endif
 
+        public const uint EXPECTED_GAME_VERSION_U = 188997904u;
+
+        public static Version ExpectedGameVersion => new Version(1, 13, 1, 1);
+
+        public static string ExpectedGameVersionString => BuildConfig.VersionToString(EXPECTED_GAME_VERSION_U, false);
+
+        public static uint CurrentGameVersionU =>
+            (uint)typeof(BuildConfig).GetField(nameof(BuildConfig.APPLICATION_VERSION)).GetValue(null);
+
+        public static Version CurrentGameVersion => new Version(
+            (int)(uint)typeof(BuildConfig).GetField(nameof(BuildConfig.APPLICATION_VERSION_A)).GetValue(null),
```

Comment:
Why do you use reflection for public const fields??

Reply:
its because I need the values at run time while constants are evaluated at compile time.

ill add comments.